commands/reshowcase

```diff
@@ -176,21 +176,27 @@ if (isBuild) {
     10
   );
 
-  const server = new WebpackDevServer(compiler, {
-    compress: true,
-    port: port,
-    publicPath: "/",
-    historyApiFallback: {
-      index: "/index.html",
+  const server = new WebpackDevServer(
+    {
+      compress: true,
+      port: port,
+      historyApiFallback: {
+        index: "/index.html",
+      },
+      devMiddleware: {
+        publicPath: "/",
+        stats: "errors-warnings",
+      },
+      ...(config.devServer || {}),
     },
-    stats: "errors-only",
-    ...(config.devServer || {}),
-  });
+    compiler
+  );
 
   ["SIGINT", "SIGTERM"].forEach((signal) => {
     process.on(signal, () => {
       if (server) {
-        server.close(() => {
+        server.stopCallback(() => {
```

Comment:
`server.close` and `server.listen` are deprecated.
`startCallback` and `stopCallback` are the new functions to use.